aiida/cmdline/commands/profile.py

```diff
@@ -43,148 +49,135 @@ def complete_profiles(self, subargs_idx, subargs):
         else:
             return ""
 
-    def profile_setdefault(self, *args):
-        from aiida.common.setup import set_default_profile
 
-        if len(args) != 1:
-            print >> sys.stderr, ("Please specify the profile to be set as the default")
-            sys.exit(1)
+@verdi_profile.command("list")
+@click.option('--no-color', is_flag=True, default=False, help='to show results without colors')
```

Comment:
Please remove this option and replace all the print statements below that use color codes manually, with the click functions. `click.secho('This will be in yellow: ', fg='yellow')` supports setting a color out of the box. Please use that in stead